src/api/TokenApi/TokenApiImpl.ts

```diff
@@ -0,0 +1,45 @@
+import { TokenApi, TokenDetails } from 'types'
+import { getTokensByNetwork } from './tokenList'
+
+const IMAGE_BASE_URL =
+  'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${name}/logo.png'
+
+function _getImageUrl(tokenAddress: string): string {
+  return IMAGE_BASE_URL.replace(/\${name}/, tokenAddress)
```

Comment:
What about using template literal rather than regex replacing? E.g:

```
getImgUrl(tokenAddress) {
  return `url....${tokenAddress}`
```

Reply:
Good idea, I thought about this, but I didn't do it cause I wanted to define the constant instead and then do the substitution in the method, and it required me to do a `eval` (i think so).

Anyways, think it's better as you suggest, so I've changed it :)